src/lib/convenience/UpdateContextAttributeRequest.h

```diff
@@ -44,6 +44,7 @@ typedef struct UpdateContextAttributeRequest
 {
   std::string                type;                // Optional
   std::string                contextValue;        // Mandatory
+  orion::ValueType           valueType;           // Type of value: either string or none
```

Comment:
Please put this new field down a few lines.
The three first fields in this struct is "the payload", after that comes the help variables, like typeFromXmlAttribute et al.